exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if (sentence.startsWith("code")){
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
The question asks you to check if a sentence contains the word "code" not if it starts with "code" You need a function that will also return true for a sentence like "I am learning to code"